### ISSUE TITLE
WCAG JSON Feedback - one tiny issue? overall very good report

### ISSUE THREAD

**Issue by airbr** [w3c/wcag#4725]
I just wanted to share an experience of using the WCAG22 JSON. I made a little app, started using the JSON, and found some formatting/content issues. I felt sort of excited that I maybe had discovered some minor issues to raise in the Repository to report. After doing a second pass at my app, I realized that the issues were all apparently on my side and the JSON is pretty much blemish-free- except maybe one tiny piece of html aria formatting to be consistent, hardly qualifies as an issue at all. I am learning an incredible amount making the little app, and clicked many links back to WAI generated from JSON. It seemed really approachable to grab the pieces of URLs like the technology ID and link it back to WAI, and it just worked.

Thank you for releasing this form of structured information. The JSON has encouraged creativity for me.

I did find one tiniest detail in my entire exploration of the JSON Document:

This is in the content section of the 2.1.2 criteria:
```
"id": "no-keyboard-trap",
"num": "2.1.2",
...
"content": "<p>If keyboard focus can be moved to a component of the page using a <a href=\"https://www.w3.org/TR/WCAG22/#dfn-keyboard-interface\" title=\"interface used by software to obtain keystroke input\">keyboard interface</a>, then focus can be moved away from that component using only a keyboard interface,\n      and, if it requires more than unmodified arrow or tab keys or other standard exit\n      methods, the user is advised of the method for moving focus away.\n   </p>\n   \n   <div class=\"note\" role=\"note\" id=\"issue-container-generatedID-17\"><div role=\"heading\" class=\"note-title marker\" id=\"h-note-17\" aria-level=\"5\"><span>Note</span></div><p class=\"\">Since any content that does not meet this success criterion can interfere with a user's\n      ability to use the whole page, all content on the web page (whether it is used to\n      meet other success criteria or not) must meet this success criterion. See <a href=\"https://www.w3.org/TR/WCAG22/#cc5\">Conformance Requirement 5: Non-Interference</a>.\n   </p></div>",
```

Specifically, in the content property value is:

`aria-level=\"5\"`

The effect of this is that inserting some of the data into my application causes it to have a level 5 heading out of order, because of the apparent aria-level. Im not sure how to override or fix this, Im sure I can fix it, it is just the only thing I found in my entire testing of the JSON document to report.

**Suggestion: remove aria level from this content value in the JSON? if it is not needed**

**Comment by bruce-usab** [comment]
@airbr thank you for your close review of the JSON and we are glad it is of utility to you!  @kfranqueiro has worked hard on sorting it out, and we have reason to believe it is used by several others.  It could be kruft, need a minor tweak, or indication of a larger issue with the file and how it‘s generated.

**Comment by airbr** [comment]
@bruce-usab Thank you @kfranqueiro it is proving very useful! The heading embedded in content seems to be in a few places in the 22 JSON where there are "Note" headings. It might be a few more places from another search of https://www.w3.org/WAI/WCAG22/wcag.json

**Comment by kfranqueiro** [comment]
This is a result of the JSON output relying directly on the TR document in its post-processed state; ReSpec adds these heading roles to all issues and notes.

I checked with Kevin and Daniel on the WAI Team, and we all agree that even within the scope of the TR document, the presence of these attributes is likely detrimental to navigability, but that'll be a broader issue to raise elsewhere later. For now, I can definitely resolve this specifically for the JSON output, since as you said it causes an even bigger problem there due to the levels not making sense out-of-context.
